au4/src/effects/builtin/amplify/AmplifyView.qml

```diff
@@ -22,6 +22,7 @@ EffectBase {
 
     Component.onCompleted: {
         amplify.init()
+        slider.value = amplify.amp
```

Comment:
In Qml, such assignment breaks the property binding
https://doc.qt.io/qt-6/qtqml-syntax-propertybinding.html#creating-property-bindings-from-javascript 

```
StyledSlider {
            value: amplify.amp
```

i.e. this bing of properties will stop working, it will be broken, we need to keep this in mind.

so we need to clearly define what we are doing:
* We initialize the initial value of the slider 
* if we move the slider, then it will simply show its current value - here it is ok
* if we change the value in another way, outside the slider, for example through the input field, then the slider will not react (the binding is broken), we need to explicitly update the slider value

Another option
* We have property binding, i.e. no matter how the property changes, the slider will react
* We need to notify about the property change at the beginning so that the initial value is read 


Now the second option is implemented. But there is some problem with the slider (

So, we can do the first option, but then we need to add an explicit update of the slider value when the amp changes.

```
    AmplifyViewModel {
        id: amplify

        onAmpChanged: {
            slider.value = amplify.amp
        }
    }
```

and remove the `value: amplify.amp` that it was not misleading (the binding is broken, this binding will not work)